packages/shared/helpers/src/number-helpers.ts

```diff
@@ -71,9 +72,10 @@ export function formatNumberAsAbbr(
     suffix = 'b';
     num = num / 1_000_000_000;
   }
+  
 
   const symbol =
-    baseCurrency && FiatSymbols[baseCurrency] ? FiatSymbols[baseCurrency] : '$';
+    baseCurrency && FiatSymbols[baseCurrency] ? FiatSymbols[baseCurrency] : hideSymbol ? '' : '$';
```

Comment:
Shouldn't the `hideSymbol` boolean come first in the ternery? If the initial condition is met then it won't work?

Reply:
This is by design since we use this function in many places and expect that it will return a $ symbol if a baseCurrency is not included.

Reply:
I can reorder it if you want

Reply:
After further investigation into the docs I don't think when can use a normal component here. A string template has to be returned. I don't think it handle JSX